src/components/Proposal/Create/SchemeForms/CreateContributionRewardProposal.tsx

```diff
@@ -145,22 +156,23 @@ class CreateContributionReward extends React.Component<IProps, IStateProps> {
     const arc = getArc();
 
     const fnDescription = () => (<span>Short description of the proposal.<ul><li>What are you proposing to do?</li><li>Why is it important?</li><li>How much will it cost the DAO?</li><li>When do you plan to deliver the work?</li></ul></span>);
+    const initialFormValues: IFormValues  = {
```

Comment:
actually now that i think about it would be better to set this up in the constructor as this.initialFormValues so it doesn't have to get recreated on every render... especially since you are loading arc for it every time. not a dealbreaker though